### PR TITLE
Dana/bugfix/buffer branch

### PR DESCRIPTION
**Pull request by daraffleur**

### CODE REVIEW

File: front/src/components/views/projects/ProjectUploadDocs.js [hunk 1]
```diff
@@ -51,6 +51,8 @@ class ProjectUploadDocs extends Component {
       project_id: this.props.match.params.id,
     };
     this.props.setProjectId(this.props.match.params.id);
+    console.log(params.project_id, "dfgfdg");
+    console.log(params.project_status, "dfgfdg");
```
Comment (Taraslysak) [inline front/src/components/views/projects/ProjectUploadDocs.js:55]: remove unnecessary logs

File: front/src/components/views/ptw/PTWToBeSubmitted.js [hunk 2]
```diff
@@ -51,14 +52,15 @@ class PTWToBeSubmitted extends Component {
   };
 
   handleCheck = (event) => {
+    console.log(this.state.workers)
```
Comment (Taraslysak) [inline front/src/components/views/ptw/PTWToBeSubmitted.js:55]: remove log

File: front/src/components/views/ptw/PTWToBeSubmitted.js [hunk 3]
```diff
@@ -51,14 +52,15 @@ class PTWToBeSubmitted extends Component {
...
     const assignedWorkers = [...this.state.workers];
     const incomingWorker = parseInt(event.target.name);
     assignedWorkers.includes(incomingWorker)
       ? assignedWorkers.splice(assignedWorkers.indexOf(incomingWorker), 1)
       : assignedWorkers.push(incomingWorker);
     this.setState({
       workers: assignedWorkers,
-    });
+    }, () => console.log(this.state.workers.length && this.state.responsible_person));
```
Comment (Taraslysak) [inline front/src/components/views/ptw/PTWToBeSubmitted.js:63]: extra log again)

**Review by Taraslysak — COMMENTED** [review]
Pretty cool, yet remove console logs)

**Review by Taraslysak — COMMENTED** [review]
look fine for me!

**Review by kipod — APPROVED** [review]

**Pull request merged**